src/_image_wrapper.cpp

```diff
@@ -86,7 +86,7 @@ _get_transform_mesh(PyObject *py_affine, npy_intp *dims)
     PyObject *output_mesh =
         PyObject_CallMethod(
             py_inverse, (char *)"transform", (char *)"O",
-            (char *)input_mesh.pyobj(), NULL);
+            (char *)input_mesh.pyobj_steal(), NULL);
```

Comment:
while you're here, can you also get rid of the casts to char\* (which are a bit weird), and also the trailing NULL (which looks just wrong, but is likely just ignored by the varargs machinery)?  also the same above.

Reply:
Probably because, pre-3.4, it was `char *`.

Reply:
Also I guess this (and above) could just use PyObject_CallMethodObjArgs, which is supposedly faster (not that it should matter here...).

Reply:
I thought that too, but unfortunately it can't because the `name` also needs to be a `PyObject*`.

Reply:
oh, well :/